src/main/resources/config/liquibase/changelog/00000000000000_initial_schema.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <databaseChangeLog
     xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
-    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:neo4j="http://www.liquibase.org/xml/ns/dbchangelog-ext"
```

Comment:
note: you can keep `ext` and use `<sql>` instead of `<neo4j:cypher>` (or `<ext:cypher>` but that looks a little strange